scripts/test.sh

```diff
@@ -541,7 +541,10 @@ function dump_deps_of_module() {
 # Checks whether dependencies are consistent across modules
 function dep_pass {
   local all_dependencies
+  local tools_mod_dependencies
+  tools_mod_dependencies=$(run_for_module "tools/mod" dump_deps_of_module "./...") || return 2
```

Comment:
Please add a comment to clarify why we need to handle tools/mod separately.

Reply:
Done :)